rocket_packet.c

```diff
@@ -13,10 +13,10 @@ unsigned int serialize_avionics_data(AvionicsData* data, uint8_t* dst) {
 	 */
 	size_t i;
 	size_t offset = 0;
-	char start_char = ROCKET_PACKET_START;
+	char start_short = ROCKET_PACKET_START;
 
 	// start char
-	memcpy(dst + offset, (void *) &start_char, sizeof(char));
+	memcpy(dst + offset, (void *) &start_short, sizeof(char));
```

Comment:
les rockets packets vont quand même juste commencer par un start char 's'
c'est les commandes envoyées au moteur qui vont avoir un start short

Reply:
It's just that the variable was a uint16_t and not a uint8_t, so I changed the name to fit the datatype.

Reply:
that's fine for the command packets, but rocket packets still have a start char, so this change should still be called char to avoid confusion